packages/slice-machine/lib/models/common/http/DefaultClient.ts

```diff
@@ -122,12 +129,15 @@ export default class DefaultClient {
     readonly repo: string,
     readonly auth: string
   ) {
-    const devConfig = (() => {
+    const devConfig: Record<string, DevConfig> = (() => {
       if (!cwd) {
         return {};
       }
       try {
-        return Files.readJson(path.join(cwd, "sm.dev.json"));
+        return Files.readJson(path.join(cwd, "sm.dev.json")) as Record<
```

Comment:
:o never heard of an sm.dev.json, do you know what it does ?

Reply:
not a clue

Reply:
this might be how @arnaudlewis or @Miniplop have solved the issue of using Slicemachine with a local instance of wroom